src/Updater.php

```diff
@@ -130,7 +136,7 @@ protected function logInfo($message) {
   /**
    * {@inheritdoc}
    */
-  public function executeUpdate($module, $update_definition_name) {
+  public function executeUpdate($module, $update_definition_name, $force = FALSE) {
```

Comment:
Can we keep adding of `force` functionality outside this PR and add that in another feature request?